doc/en/components/grids/_shared/row-pinning.md

```diff
@@ -235,11 +235,12 @@ grid.pinning = { rows: RowPinningPosition.Bottom };
 
 <!-- ComponentStart: Grid, TreeGrid, HierarchicalGrid -->
 ```tsx
-<{ComponentSelector} id="dataGrid" autoGenerate="true">
+<{ComponentSelector} ref={gridRef} autoGenerate={true}>
 </{ComponentSelector}>
 
-var grid = document.getElementById("dataGrid") as {ComponentSelector};
-grid.pinning = { rows: RowPinningPosition.Bottom };
+const pinning: IgrPinningConfig = { rows : RowPinningPosition.Bottom }; 
+
+gridRef.current.pinning = { pinning }
```

Comment:
This is not how u use that `as input`. You should use it `<IgrGrid pinning={pinning}>`